### DISCUSSION TITLE
Where to put per-user settings ?

### DISCUSSION THREAD

**Discussion by guillaumecherel** [microsoft/pyright#3211] [Q&A]
Hi,

The doc mentions that the configuration for "venv" and "venvPath" can be specified in a per-user setting. How can I do this ?

Thanks !

**Comment by erictraut** [comment]
The "venvPath" is intended to be per-user, but the "venv" name is meant to be shared among all users.

The "venvPath" can be set through the LSP using the setting "pyright.venvPath" or on the command line using the "--venv-path" option.
